content/contributing/documentation.rst

```diff
@@ -132,12 +132,16 @@ to make changes from the GitHub interface.
 
    .. tabs::
 
-      .. group-tab:: Linux and macOS
```

Comment:
This is not the end of the world, but the tab sync between this group and the other "Linux and macOS" groups will stop working. We can merge like that, but it could be worth looking for a way to keep them in sync, like referencing a guide that covers both systems.

Reply:
> This is not the end of the world, but the tab sync between this group and the other "Linux and macOS" groups will stop working.

@AntoineVDV  I looked for an existing article or guide on installing Make on Linux and macOS, but I did not find anything that covered both systems effectively.

Maybe to keep the tab sync, we can list the OS like this?

<img width="414" alt="Screenshot 2025-02-21 at 10 35 35 AM" src="https://github.com/user-attachments/assets/5ad7115c-627e-4af9-b156-0dd016550d72" />


```rst
      .. group-tab:: Linux and macOS

         **Linux**

         .. code-block:: console

            $ sudo apt install make -y

         **macOS**

         .. code-block:: console

            $ sudo apt install make -y
```

Reply:
No, that would defeat the purpose of the tabs. Let's split this group then, and if we find that it's too weird, we can still split all the "Linux and macOS" groups so that they're synced again.